src/components/editTable.vue

```diff
@@ -15,7 +15,7 @@
       <slot name="tbody">
         <tr v-for="(data, index) in tabledata" :key="index">
           <template v-for="(column, idx) in columns">
-            <td v-if="!column.field_name.includes('id')" :key="column.field_name + idx" :class="idx">
```

Comment:
I see a lot of removing the key, is this recommended from vue. I know in vue 2 the key was used for tracking updates so as to not re-render more than necessary.